satellite/admin-back-office-v1.md

```diff
@@ -0,0 +1,554 @@
+# Admin Back Office v1
+
+Considerations when reading this document.
+- We use the terms "web interface" and "API" for referring to each part of the system and we use
+  "back office" when we want to refer to the system as a whole.
+- We use the term "operator" to refer to people who has access to the back office.
+
+## Essentials
+
+### Header
+
+Date: 2023-07-27
+Owner: Ivan Fraixedes
+Accountable: TBD
+Consulted:
+- Product. Reference person [Andi Steinke](https://github.com/stoweandi)
+- Customer support team. Reference person [Helene Unland](https://github.com/heunland).
+- Production owners.
+- Team integrations console squad: [Maximillian von Briesen (Moby)](https://github.com/mobyvb).
+Informed:
+- QA team.
+
+### Context
+
+Satellite currently has an [administration
+API](https://github.com/storj/storj/blob/main/satellite/admin/README.md)
+and minimalist [Web interface](https://github.com/storj/storj/tree/main/satellite/admin/ui) which
+allows to interact with its endpoints through Web forms.
+
+API authentication is conducted by two mechanism:
+- [An unique token secret](https://github.com/storj/storj/blob/main/satellite/admin/server.go#L215).
+  The service isn't exposed publicly, the only way to access to it is forwarding the port of the
+  corresponding Kubernetes pod.
+- [Allowed groups](https://github.com/storj/storj/blob/main/satellite/admin/server.go#L231).
+  This requires that the server is accessed through an authorization proxy (e.g.
+  [Oauth2 proxy](https://oauth2-proxy.github.io/)):
+  - AP1: https://ap1.admin.storj.tools
```

Comment:
Doesn't it make more sense to keep this on `storj.io`, because it's critical infrastructure?

Reply:
Perhaps, but this is detailing the current status and they are served under storj.tools domain

Reply:
Do you think that we should clarify in this document which domain should serve the new back office?

Reply:
If this is a design doc, then yes, if this documents what the current behavior is, then no.

Reply:
this document is a design doc for the new back office, this section is the context that describes the current system.

Reply:
Then I would probably mention it in some other section.